tests/foreman/ui/test_location.py

```diff
@@ -168,12 +184,15 @@ def test_positive_update_with_different_names(self, new_name):
         @assert: Location name is updated
 
         """
-        loc_name = gen_string('alpha')
         with Session(self.browser) as session:
+            loc_name = gen_string('alpha')
```

Comment:
Move it outside the `Session` as before (we discussed this today and agreed to do it)